crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -328,6 +328,48 @@ where
                     guard.on_chain_sync_interval
                 });
 
+                let channels = match node_storage.all_channels_without_costs() {
+                    Ok(channels) => channels,
+                    Err(e) => {
+                        tracing::error!("Failed to fetch all channels without costs. Error: {e:#}");
+                        vec![]
```

Comment:
🔧 I think `continue` is more idiomatic here.

Reply:
Hmm, I assume the sleep is (or should be) outside of the loop. `continue` just goes to the next iteration.

Reply:
Not sure if I get you, the sleep wouldn't make much sense outside of the loop, would it?